src/test/resources/schemas/isPartOf.json

```diff
@@ -1,37 +1,40 @@
 {
-    "$id": "isPartOf.json",
-    "$schema": "http://json-schema.org/draft-07/schema#",
-    "title": "Is part of",
-    "description": "The isPartOf relation",
-    "type": "array",
-    "items": {
-        "type": "object",
-        "properties": {
-            "type": {
-                "type": "array",
-                "items": {
-                    "type": "string",
-                    "const": "IsPartOfRelation"
-                }
-            },
-            "numbering": {
-                "type": "string"
+  "$id": "isPartOf.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
indentation problem

Reply:
It seems that we are not following our own rules here. JSON should have 2 spaces as identation. See:

https://github.com/hbz/lobid-resources/blob/8e9700fdc6e860d7a50c106d769d7150e2e1bfec/.editorconfig#L11-L13

Reply:
ok